src/components/transactions/TxDetails/TxData/DecodedData/MethodCall.tsx

```diff
@@ -0,0 +1,55 @@
+import NamedAddressInfo from '@/components/common/NamedAddressInfo'
+import { Box, Typography } from '@mui/material'
+
+const MethodCall = ({
+  method,
+  contractAddress,
+  contractName,
+  contractLogo,
+}: {
+  method: string
+  contractAddress: string
+  contractName?: string
+  contractLogo?: string
+}) => {
+  return (
+    <>
+      <Typography fontWeight="bold" display="flex" alignItems="center" gap=".5em" pb={1.5} component="div">
+        Call{' '}
```

Comment:
I suggest we change the display to block or similar to make it readable on mobile:

<img width="374" alt="Screenshot 2024-07-22 at 12 17 10" src="https://github.com/user-attachments/assets/82581bfa-68c1-46f6-ace0-6a7101d73c57">